src/routes/threads/[id]/+page.svelte

```diff
@@ -0,0 +1,271 @@
+<script lang="ts">
+    import { Main } from '$lib/layouts';
+    import { DEFAULT_DESCRIPTION } from '$lib/utils/metadata';
+    import { TITLE_SUFFIX } from '$routes/titles';
+
+    import FooterNav from '$lib/components/FooterNav.svelte';
+    import MainFooter from '$lib/components/MainFooter.svelte';
+    import SeoOgImage from '$lib/components/SeoOgImage.svelte';
+    import PreFooter from '../PreFooter.svelte';
+    import MessageCard from './MessageCard.svelte';
+
+    export let data;
+
+    const title = 'Support Thread' + TITLE_SUFFIX;
+    const description = DEFAULT_DESCRIPTION;
+    const discordLink = "https://discord.com/channels/564160730845151244/{data.discord_id}";
```

Comment:
This should be a template string, currently it will always link to [https://discord.com/channels/564160730845151244/{data.discord_id}](https://discord.com/channels/564160730845151244/{data.discord_id})